docs/getting-started-guides/clc.md

```diff
@@ -316,13 +322,14 @@ kubectl proxy -p 8001
 
 Then, you can access urls like ```http://127.0.0.1:8001/api/v1/proxy/namespaces/kube-system/services/kube-ui/``` without the need for client certificates in your browser.
 
+## Load Balancer integration.
 
+Our Kubernetes code includes definitions of CenturyLinkCloud as a Kubernetes cloudprovider, which includes integration of the CLC Load Balancer services. When a Kubernetes service is defined as type: LoadBalancer, a public ip address is automatically obtained and mapped to the service endpoint.
```

Comment:
Our Kubernetes code includes definitions of CenturyLink Cloud as a provider, which includes integration of the CLC Load Balancer services. When a Kubernetes service is defined a `LoadBalancer`, a public IP address is automatically obtained and mapped to the service endpoint.